app/templates/teachers/create-teacher-account-view.jade

```diff
@@ -0,0 +1,166 @@
+extends /templates/base-flat
+
+block content
+  .container
+    form
+      .row.text-center
+        .col-md-offset-2.col-md-8
+          h3.m-t-3(data-i18n="teachers_quote.create_account")
+          h4(data-i18n="[html]teachers_quote.create_account_subtitle")
+          .alert.alert-info.m-y-2
+            span.spr(data-i18n="signup.login_switch")
+            a.login-link(data-i18n="login.log_in")
+          #social-network-signups.m-y-2
+            span.spr(data-i18n="teachers_quote.signup_with")
+            button#facebook-signup-btn.btn.btn-navy.btn-lg.m-x-1 Facebook
+            button#gplus-signup-btn.btn.btn-burgandy.btn-lg.spr G+
```

Comment:
These are not Facebook or Google+ login buttons.  They appear to be boostrap buttons.
https://developers.facebook.com/docs/facebook-login/web/login-button
https://developers.google.com/identity/sign-in/web/sign-in#add_a_google_sign-in_button